cluster/common.sh

```diff
@@ -682,6 +682,21 @@ EOF
 INITIAL_ETCD_CLUSTER: $(yaml-quote ${INITIAL_ETCD_CLUSTER})
 EOF
     fi
+    if [ -n "${APISERVER_FEATURE_CONFIG:-}" ]; then
+      cat >>$file <<EOF
+APISERVER_FEATURE_CONFIG: $(yaml-quote ${APISERVER_FEATURE_CONFIG})
```

Comment:
Same set of features for all components sgtm.

As for the version lag problem, there is no good way to resolve with the proposed mechanism (per-component flag). If a feature requires activation on both master and node components, it's up to the implementer to pick sane defaults and behavior if one is on and the other not. Or to say that the feature doesn't support version skew (in which case it should almost certainly be off by default). This is no different than the current state of affairs, except we'll now have explicit feature activation in command line, instead of it being buried somewhere deep in the code.